### DISCUSSION TITLE
[VOTE] Release Apollo Java version 2.1.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo-java#19] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo Java version 2.1.0.

Release Note:

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.1.0

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1035/content/com/ctrip/framework/apollo/

You may add the following to pom.xml to test the new artifacts.

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1035/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.1.0

Release CommitID :

 * https://github.com/apolloconfig/apollo-java/commit/9ea2213ff34fb8a0189e5aeb611f890b71940453

Guide to build the release from source:

 * mvn clean package -DskipTests=true

Voting will start now (2023/01/28) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by klboke** [comment]
+1 ,I checked the list of release features and there is no problem

**Comment by Accelerator96** [comment]
+1

**Comment by zouyx** [comment]
+1 accepted

**Comment by kezhenxu94** [comment]
My +1 to release Apollo Java 2.1.0

**Comment by JaredTan95** [comment]
+1 for Apollo Java 2.1.0

**Comment by Anilople** [comment]
+1

---

Through v2.1.0's newest commit id is f8d28868e5c300bca0161885a316cce3639bd342 not 9ea2213ff34fb8a0189e5aeb611f890b71940453
but I think it is ok because f8d28868e5c300bca0161885a316cce3639bd342 was only change the release.yml

**Comment by nisiyong** [comment]
+1 for Apollo Java 2.1.0 newest commitId `f8d2886`. Do we need to update the release description and redeploy?

**Reply by nobodyiam** [reply]
The latest commit https://github.com/apolloconfig/apollo-java/commit/f8d28868e5c300bca0161885a316cce3639bd342 only changed the release workflow file, which doesn't affect the release description and the maven artifacts. So I think it's ok to leave it as is.

**Comment by nobodyiam** [comment]
The vote to release Apollo Java version 2.1.0 has passed with seven +1 and zero -1.
